CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 ### Fixes
 
+* **Fix broken API unit test** where the test was using `fast` stragy for images, which is not supported.
```

Comment:
can remove this since it does not impact the library. we can force merge if there is a changelog complaint.